app/services/foreman_ansible/insights_plan_runner.rb

```diff
@@ -27,10 +27,8 @@ def run_playbook
     # Fetches the playbook from the Red Hat Insights API
     def playbook
       resource = RestClient::Resource.new(
-        'https://api.access.redhat.com/r/insights/'\
-        "v3/maintenance/#{@plan_id}/playbook",
-        get_ssl_options_for_org(@organization, nil).
-        merge(:verify_ssl => OpenSSL::SSL::VERIFY_NONE)
+        "#{insights_api_host}/r/insights/v3/maintenance/#{@plan_id}/playbook",
```

Comment:
@lphiri Where is the `insights_api_host` variable coming from?

Reply:
@dLobatog  its a method from the "RedhatAccess::Telemetry::LookUps" module - the same place you are getting " get_ssl_options_for_org".